__tests__/hooks/useTheme.spec.js

```diff
@@ -0,0 +1,29 @@
+/* globals expect, test */
+import { mount } from 'enzyme'
+import React from 'react'
+import {
+  Provider,
+  useTheme
+} from '../../dist'
+
+const Sample = () => {
+  const theme = useTheme()
+
+  return (
+    <span>{theme.foo}</span>
+  )
+}
+
+test.only('useTheme should return correct theme object', () => {
```

Comment:
`.only` :police_car: 
```suggestion
test.only('useTheme should return correct theme object', () => {
```